lib/huiClient.js

```diff
@@ -0,0 +1,47 @@
+const log = require('@financial-times/n-logger').default;
+
+const huiBasePath = process.env.HUI_BASE_PATH;
+const licencePath = '/licences/';
+const options = {
```

Comment:
If you're going to use `fetch-retry-or-die` (as mentioned below). You might want to extend `config.fetchOptions` (as all the other client proxies do - https://github.com/Financial-Times/kat-client-proxies/blob/master/lib/accessLicenceClient.js#L11)